content/2023-08-29_pyladies-byly-jsme-na-europythonu.md

```diff
@@ -0,0 +1,57 @@
+Title: PyLadies & Pyvo - Byly jsme na EuroPythonu
+Date: 2023-08-29 18:00:00
+Author: Karolina Surma, Mia Bajic
+
+Do Prahy letos poprvé doputovala konference EuroPython.
+Mezi 17. 7. a 23. 7. 2023 jste tam mohli potkat velkou část české Python komunity.
+EuroPython nám poskytl jak možnost posílit vazby v naší komunitě, tak rozšířit sítě do mezinárodních vod.
+
+S PyLadies a s Pyvem jsme to nemohly zmeškat!
+
+![samolepky_pyladies]({static}/images/samolepky_pyladies_2023.jpg)
+
+
+## Aktivity PyLadies
+
+S nápadem, že oslovíme organizátory EuroPythonu, přišla **Monika Bláhová** z brněnských PyLadies.
+Vděčíme jí za rozhýbání a finální rozsah prezentace PyLadies na EuroPythonu.
+Původně byl v plánu pouze večerní sraz PyLadies a příznivců, z něhož se postupně vyklubaly další aktivity.
+Realizace dopadla na výbornou díky nasazení, jaké do příprav vložila **Mia Bajić**. Díky!
+
+![aktivity_Python_komunity_na_EuPy]({static}/images/czech_python_at_eupy.jpg)
+
+
+### Hadí workshop
+
+**17. 7. 2023**, hned na začátku konference, se konal celodenní [Hadí workshop](https://ep2023.europython.eu/pyladies-workshop)
+(zmiňován již zde na blogu [v roce 2019](https://blog.python.cz/pyladies-snake-report) a [2020](https://blog.python.cz/Hadi-workshop-v-Brne]).
+Ovšem letos to bylo poprvé **anglicky**.
+Workshop opět nachystal a bravurně přednesl **Petr Viktorin** s výpomocí party **kouček a koučů** z Prahy, Plzně a Brna.
+Někteří s námi koučovali vůbec poprvé, vítáme v komunitě :)
+
+![snake_workshop]({static}/images/snake-workshop-eupy-2023.jpg)
+
+### PyLadies Social Event
+
+**21. 7. 2023**, tentokrát na zakončení konference, jsme pozvaly všechny PyLadies a příznivce na [večerní společenský event](https://ep2023.europython.eu/pyladies-social-event).
+Do Hospůdky na Hradbách dorazilo přes 80 účastnic a účastníků konference.
+Akce sklidila velký úspěch.
+
+## Pyvo
+V Praze už přes 10 let každou třetí středu v měsíci probíhá pražské Pyvo a EuroPython se konal přesně v tomto týdnu.
+Ve spolupráci se pořadateli EuroPythonu jsme naplánovaly společné Pyvo, abychom daly naší místní komunitě příležitost pokecat s účastníky EuroPythonu.
+Na Pyvo přišlo kolem 150 lidí a bylo to zatím největší a organizačně nejnáročnější Pyvo, co jsme v našich dějinách organizovaly.
+Jsem velmi rády, že se nám podařilo vytvořit skvělý program a dát místní komunitě příležitost potkat se s lidmi z celého světa, a zároveň jsme udělaly skvělé promo české komunity pro lidi ze zahraničí.
```

Comment:
```suggestion
Jsem velmi rády, že se nám podařilo vytvořit skvělý program a dát místní komunitě příležitost potkat se s lidmi z celého světa a zároveň jsme udělaly skvělé promo české komunity pro lidi ze zahraničí.
```

Reply:
Zde mi přijde, že ta čárka je nutná

Reply:
Souhlas: https://prirucka.ujc.cas.cz/?id=153#nadpis4

Jen bych opravil to `Jsem` na `Jsme` :)

Reply:
Myslel jsem: Jsme rády, že A a B.

Reply:
S tím, že _a zároveň_ je slučovací.

Reply:
Hlavní otázka asi je: chceš říct, že jste rády, že jste udělali promo? Nebo jste rádi patří jen k první části? V druhém případě bych začal Zároveň novou větu:

```suggestion
Jsme velmi rády, že se nám podařilo vytvořit skvělý program a dát místní komunitě příležitost potkat se s lidmi z celého světa. Zároveň jsme udělaly skvělé promo české komunity pro lidi ze zahraničí.
```

Reply:
Díky, předěláno na dvě věty v dcf5b3b37b4b6d84c8d6a7359992b37051d0276a